modules/nextflow/src/main/groovy/nextflow/k8s/model/JobSpecBuilder.groovy

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2020-2022, Seqera Labs
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.k8s.model
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+
+/**
+ * Object build for a K8s job specification
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@CompileStatic
+@Slf4j
+class JobSpecBuilder {
+
+    String jobName
+
+    String namespace
+
+    String workDir
```

Comment:
We may need to review the need to use a workDir. This has been recently removed for the pod execution. See https://github.com/nextflow-io/nextflow/commit/dc8bfeadc46f82eaadb192833f54efcb6b5bc582

Reply:
Ok, we can remove workDir at all.

Reply:
fixed